core/primitives/block/block.cpp

```diff
@@ -0,0 +1,6 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "primitives/block/block.hpp"
```

Comment:
Is it needed?

Reply:
[> Pls consider adding a library for block linking all the dependencies to it, and other minor issues.](https://github.com/filecoin-project/cpp-filecoin/pull/68#pullrequestreview-343509730)